test/unit/Dockerfile

```diff
@@ -4,10 +4,9 @@ MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 ENV FLUENTD_VERSION=1.0 \
     WORKDIR=/fluentd/lib
 
-WORKDIR ${WORKDIR}
-
 ENTRYPOINT /tmp/run.sh
 RUN scl enable rh-ruby25 -- gem install bundler --no-ri --no-doc
+WORKDIR ${WORKDIR}
```

Comment:
using this dockerfile, running from the oal directory:
```
imagebuilder -t test:test -f test/unit/Dockerfile test/unit/
--> FROM centos/ruby-25-centos7:latest as 0
--> MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
--> ENV FLUENTD_VERSION=1.0     WORKDIR=/fluentd/lib
--> ENTRYPOINT /tmp/run.sh
--> RUN scl enable rh-ruby25 -- gem install bundler --no-ri --no-doc
Successfully installed bundler-2.0.2
1 gem installed
--> WORKDIR ${WORKDIR}
--> COPY test/unit/run.sh /tmp/
stat test/unit/test/unit/run.sh: no such file or directory
```
similar result with `docker build`

Reply:
doing a `cd test/unit` first:
```
imagebuilder -t test:test .
--> FROM centos/ruby-25-centos7:latest as 0
--> MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
--> ENV FLUENTD_VERSION=1.0     WORKDIR=/fluentd/lib
--> ENTRYPOINT /tmp/run.sh
--> RUN scl enable rh-ruby25 -- gem install bundler --no-ri --no-doc
Successfully installed bundler-2.0.2
1 gem installed
--> WORKDIR ${WORKDIR}
--> COPY test/unit/run.sh /tmp/
stat test/unit/run.sh: no such file or directory
```
same result with `docker build`

Reply:
This works locally:
```$ podman build -f test/unit/Dockerfile -t test-unit:test-unit .
STEP 1: FROM centos/ruby-25-centos7:latest
Getting image source signatures
Copying blob ee720ba20823 done
Copying blob 8ba884070f61 done
Copying blob c3dca185eb14 done
Copying blob 2e1bca9ede03 done
Copying blob 497ef6ea0fac done
Copying blob ebf1fb961f61 done
Copying blob 8ca7d9bd539c done
Copying blob fa64bfcb1a7d done
Copying blob 80d8b93d3338 done
Copying config cb490f3564 done
Writing manifest to image destination
Storing signatures
STEP 2: MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
6e52f2e1d984db410211f87fd7fb95a62f9deb3b2427ec64183313a5d6f63e58
STEP 3: ENV FLUENTD_VERSION=1.0     WORKDIR=/fluentd/lib
586b043a7f9d1190046a6a47ab070cb3c0512bad91d0534625b3af467fa2a298
STEP 4: ENTRYPOINT /tmp/run.sh
45e97051406e68878ae78f25b28904bd9d48593b26c2bf74553290570aeaa950
STEP 5: RUN scl enable rh-ruby25 -- gem install bundler --no-ri --no-doc
Fetching: bundler-2.0.2.gem (100%)
Successfully installed bundler-2.0.2
1 gem installed
8748bc328a7efd5469701f8feb7181d8551b1dcb43b4b1e11a57d1f11cf64962
STEP 6: WORKDIR ${WORKDIR}
97985b9f161920b4cf5c4384785b288521250e9936856c5215c589ff11eff60e
STEP 7: COPY test/unit/run.sh /tmp/
c5f0d93918621efb559103d36183ec940e53859247766bbf992b4823cb3016f0
STEP 8: COPY fluentd/lib ${WORKDIR}
a3c22c79a0cc66944efa3ba2efcf5f3eebcd0945b6e7ea80f31ede3a074b6462
STEP 9: RUN for d in $(ls $WORKDIR) ; do     pushd ${d} ;         export GEM_HOME=vendor ;         scl enable rh-ruby25 -- bundle install ;     popd ;     done
/fluentd/lib/filter_elasticsearch_genid_ext /fluentd/lib
Fetching gem metadata from https://rubygems.org/.........
```